### DISCUSSION TITLE
feature: use markdownlint-cli2 instead of markdownlint

### DISCUSSION THREAD

**Discussion by rofrol** [LazyVim/LazyVim#3479] [Extra Requests]
### Did you check the docs?

- [X] I have read all the LazyVim docs

### Is your feature request related to a problem? Please describe.

https://github.com/DavidAnson/markdownlint-cli2 is from the same author of https://github.com/DavidAnson/markdownlint.

Currently used is https://github.com/igorshubovych/markdownlint-cli.

Here is PR with info that markdownlint-cli2 is better https://github.com/stevearc/conform.nvim/pull/171

### Describe the solution you'd like

Update markdown.lua to use markdownlint-cli2

### Describe alternatives you've considered

None

**Comment by rubiin** [comment]
Further reads https://dlaa.me/blog/post/markdownlintcli2
Btw you can create a pr if you like just changing the references of markdownlint to markdownlint2 should be enough imo
https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/extras/lang/markdown.lua
